Firestore/Source/Util/FSTLogger.m

```diff
@@ -19,22 +19,33 @@
 #import <FirebaseCore/FIRLogger.h>
 
 #import "Firestore/Source/API/FIRFirestore+Internal.h"
+#import "Firestore/Source/API/FIRFirestoreVersion.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+// assumes the first variable argument is the sdk version, and adds a space for it to the format
+// string.
+void logInternal(FIRLoggerLevel level, NSString* format, ...) {
+  NSString* formatWithVersion = [@"v%s - " stringByAppendingString:format];
```

Comment:
The version should also be logged by the C++ implementations in 

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/util/log_apple.mm

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/util/log_stdio.cc